File: Walter/LENGUAJE_ESTRUCTURADO_12062023-main/APACHENETBEANS/src/poo/circulo/Principal.java
package poo.circulo;

import java.util.Scanner;

public class Principal {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;
        Circulo c;
        System.out.print("Cantidad objetos circulo? ");
        n = sc.nextInt();
        Circulo[] circulos = new Circulo[n];
        //LLENAR
        for(int i=0; i<n; i++) {
            circulos[i] = new Circulo(Math.random()*100+1);
        }
        //MOSTRAR
        Circulo.cabecera();
        for(int i=0; i<n; i++) {
            c = circulos[i];
            c.cuerpo(i);
        }
      
    }
    
}

File: Walter/Python_walter/POO/Ejemplo2/principal.py
import os
from circulo import Circulo
import random

if __name__ == "__main__":
   os.system('cls')
   Circulo.cabecera()
   for i in range(100):
       radio = random.uniform(1,100)
       c = Circulo(radio)
       c.cuerpo(i+1)
